tables/atom.py

```diff
@@ -1092,7 +1097,8 @@ class VLStringAtom(_BufferedAtom):
     base = UInt8Atom()
 
     def _tobuffer(self, object_):
-        if not isinstance(object_, basestring):
+        if not isinstance(object_, bytes) and \
+                not isinstance(object_, unicode):
```

Comment:
I thought we wanted to issue a `DeprecationWarning` when someone is trying to feed a `VLString` with a `unicode` string.  I am missing sometthing?